### DISCUSSION TITLE
How can I order issue templates?

### DISCUSSION THREAD

**Discussion by bmunkholm** [community/community#21601] [New to GitHub]
<p>Is there a way to ensure a specific order of the issue templates when a user creates a new issue?<br>
And also get the “links” listed in config.yml up about the templates?</p>

**Comment by kingthorin** [comment]
<p>They’re ordered alphabetically by file name. So you can take advantage of that including using numbers (zero padded if you have more than 10).</p>

**Comment by bmunkholm** [comment]
<p>Thanks - that works for the templates.<br>
But not for the “links” in config.yml. Is there a way to get the “links” before the templates? ( I tried calling templates “x1.md”, but I still get templates before “links”.</p>

**Comment by kingthorin** [comment, marked as answer]
<p>As far as I know that’s just how the GitHub web UI is designed, I don’t think you can control it.</p>

**Comment by bmunkholm** [comment]
<p>Thanks for your help!</p>

**Comment by szaimen** [comment]
<p>Would be great to have this feature implemented. Then you could place a link with FAQ over any issue templateee which would be quite useful, imo</p>

**Comment by dluc** [comment]
Allowing to control where links are positioned is really necessary, to lead users through the appropriate path, also considering that GitHub injects other links like "Report a security vulnerability" before the custom links. 

For instance, about "Report a security vulnerability", pushing custom links further down means that most users create an issue instead of starting a discussion.  This causes unnecessary maintenance and duplication, such as the need to convert issues to discussions, which creates new IDs and generates email notifications.
